### PR TITLE
Added solutions to Near Me 2

### PR DESCRIPTION
**Pull request by supermajda**

### CODE REVIEW

File: Web Developer Skills/Learn AngularJS/Unit 5/NearMe 2/MainController.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+app.controller('MainController', ['$scope', 'places', function($scope, places){
+
+places.success(function(data){
```
Comment (ummahusla) [inline Web Developer Skills/Learn AngularJS/Unit 5/NearMe 2/MainController.js]: Can we have a proper formatting here

File: Web Developer Skills/Learn AngularJS/Unit 5/NearMe 2/MainController.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+        $scope.geodata = data;
+     $scope.mapMarkers = geodataToMarkers($scope.geodata);
+
+  });
+
+  $scope.mapCenter = 
+    {
```
Comment (ummahusla) [inline Web Developer Skills/Learn AngularJS/Unit 5/NearMe 2/MainController.js]: Formatting required here as well

File: Web Developer Skills/Learn AngularJS/Unit 5/NearMe 2/places.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+app.factory('places', ['$http', function($http) {
+  return $http.jsonp('https://en.wikipedia.org/w/api.php?action=query&list=geosearch&gsradius=5000&gscoord=40.741934%7C-74.004897&gslimit=30&format=json&callback=JSON_CALLBACK')
+    .success(function(data) {
```
Comment (ummahusla) [inline Web Developer Skills/Learn AngularJS/Unit 5/NearMe 2/places.js]: Formatting required here as well

File: Web Developer Skills/Learn AngularJS/Unit 5/NearMe 2/places.js [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+    return data;
+    })
+        .error(function(err) {
```
Comment (ummahusla) [inline Web Developer Skills/Learn AngularJS/Unit 5/NearMe 2/places.js]: Formatting required here as well

**Comment by ummahusla** [comment]
@supermajda Hey, congrats on your first PR :tada: Everything seems to be fine, just a lack of the code formatting. In case you not sure how is better to format it, you can look into [**Prettier**](https://github.com/prettier/prettier). It's a javascript code formatter.

**Comment by supermajda** [comment]
Hey @ummahusla - thanks so much! I just reformatted - please take a look. If its all good, I'll start adding solutions to other excersizes and pay more attention to formatting :)

**Comment by ummahusla** [comment]
@supermajda Thank you for your effort 👍

**Pull request merged**